scripts/clean-build.sh

```diff
@@ -36,6 +36,8 @@ apt-get install -qq -y \
   pkg-config \
   protobuf-compiler
 
+apt-get clean
```

Comment:
It is but we only create an ephemeral image here which isn't persisted anywhere so an `apt-get clean` will not have any effect. If we would have a `Dockerfile` which prepares a docker container for us with all necessary things preinstalled then it would but in this case it doesn't.